api/v1beta1/kafkacluster_types.go

```diff
@@ -407,6 +428,8 @@ type IngressServiceSettings struct {
 	// In case of external listeners using NodePort access method the broker instead of node public IP (see "brokerConfig.nodePortExternalIP")
 	// is advertised on the address having the following format: <kafka-cluster-name>-<broker-id>.<namespace><value-specified-in-hostnameOverride-field>
 	HostnameOverride string `json:"hostnameOverride,omitempty"`
+	// Template used to generate broker hostnames for tls enabled envoy. %id will be replaced with brokerId value
+	BrokerHostnameTemplate string `json:"brokerHostnameTemplate,omitempty"`
```

Comment:
@dobrerazvan can this be moved to KafkaClusterSpec.EnvoyConfig ?

The reason for this is that the comment says  it applies to "envoy" ingress while this `IngressServiceSettings` is generic for all kafka ingresses (istio, envoy)

Reply:
@amuraru Implemented the review.